### DISCUSSION TITLE
[UI & MultiLanguage]Chinese Translations' deviation

### DISCUSSION THREAD

**Discussion by Shanshui2024** [community/community#145733] [Mobile] [answered]
As the title suggests, there is a deviation in the Chinese translation of Github Mobile.
For example, Issues may be translated as (political) topics, but in reality it should not be so.
For example, some specific terms, such as Pull Requests, will be translated as “请求提交” , which means “Submit Request”.
If there is a need, can your company contact Chinese netizens or developers for translation?Because similar translation issues can lead to misunderstandings (although we can also understand terms like Pull Requests and Issues).

Thanks.

**Comment by HenSquared** [comment, marked as answer]
Hey thanks for the feedback - I've created an internal issue to track this.
